frontend/app/utils/LegacyHashing.java

```diff
@@ -0,0 +1,21 @@
+package utils;
+
+import com.google.common.hash.HashFunction;
+import com.google.common.hash.Hashing;
+
+/**
+ * Provides an md5 and sha1 hash function without producing deprecation warnings.
+ * From: https://github.com/google/guava/issues/2841
+ */
+
+public class LegacyHashing {
```

Comment:
does this need to be a class with just static methods or can it just be an object?

Reply:
I don't think you can suppress warnings on individual lines in Scala, so this file has to be Java

Reply:
Oh, somehow  I didn't even notice it was java.